app/views/static_pages/about.html.slim

```diff
@@ -0,0 +1,91 @@
+.page-details--creme-bg
+  .container
+    = render 'shared/alerts'
+    .row
+      .col-xs-12
+        .form-box
+          .form-box__title
+            h2 About the Commons
+          .form-box__body
+            p
+              | The Commons is a knowledge-sharing, learning platform for environmental research and action.
+              | It provides a place for people working on environmental issues to
+              = ' '
+              span.form-box__underline find and share
+              = ' '
+              | resources, information, and ideas;
+              = ' to '
+              span.form-box__underline build
+              = ' '
+              | networks; and to
+              = ' '
+              span.form-box__underline work collaboratively
+              = ' '
+              | . We aim to provide users the information and tools that will help them create a
+              | more sustainable world.
+
+            h5 Vision
```

Comment:
Can we add a bit of margin above the h5? Also, can you make it a bit bigger? Perhaps an h4 or even h3?